datalad/core/distributed/tests/test_clone.py

```diff
@@ -910,3 +910,19 @@ def test_clone_unborn_head_sub(path):
     ds_cloned_sub = ds_cloned.get(
         "sub", result_xfm="datasets", return_type="item-or-list")
     eq_(ds_cloned_sub.repo.get_active_branch(), "other")
+
+
+@skip_if_no_network
+@with_tempfile
+def test_gin_cloning(path):
+    # can we clone a public ds anoynmously from gin and retrieve content
+    ds = clone('https://gin.g-node.org/datalad/datalad-ci-target', path)
+    ok_(ds.is_installed())
+    ok_(ds.repo.file_has_content('annex/two') is False)
```

Comment:
Instead of `ok_(... is False)` I'd personally prefer to use `eq_(..., False)`, but there is also `nok_()`.

Reply:
done